### PR TITLE
Add scripts to rule them all

### PR DESCRIPTION
**Pull request by pezholio**
This adds scripts using the [Scripts to Rule Them All](https://github.com/github/scripts-to-rule-them-all) pattern to make it easier to get started with the app.

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+worker: bundle exec sidekiq -C config/sidekiq.yml
```
Comment (leeky) [inline Procfile:1]: NB: There aren't currently any jobs for the worker to run and I'm not actually sure *when* we'll need Sidekiq.

File: script/console [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      echo "Unknown environment '$ENVIRONMENT'"
+      exit 1
+      ;;
+  esac
+else
+  echo "==> Updating..."
+  script/update
+
+  echo "==> Starting a local Rails console..."
+  bundle exec rails console
```
Comment (leeky) [inline script/console:24]: Is it worth adding `--sandbox` support to this?

File: script/test [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+if [ -n "$TEST_FILE" ]; then
+  echo "==> Running the tests matching '$TEST_FILE'..."
+  bundle exec rspec --pattern "$TEST_FILE"
+else
+  if [ -n "$CI" ]; then
+    echo "==> Linting Ruby..."
+    bundle exec standardrb
+  else
+    echo "==> Linting Ruby in fix mode..."
+    bundle exec standardrb --fix
```
Comment (leeky) [inline script/test:30]: This is nice! :+1:

File: script/console (hunk 2 above)
Comment (pezholio) [reply script/console:24]: Oooh, good plan 👍

File: script/test (hunk 3 above)
Comment (pezholio) [reply script/test:30]: I nicked it off the Rails template, so can't claim credit 😄

File: Procfile (hunk 1 above)
Comment (pezholio) [reply Procfile:1]: Ooooh. Is it worth keeping this just in case?

File: script/update [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+cd "$(dirname "$0")/.."
+
+echo "==> Bootstrapping..."
+script/bootstrap
+
+echo "==> Running database migrations..."
+bundle exec rails db:migrate
+
+echo "==> Running test database migrations..."
+RAILS_ENV="test" bundle exec rails db:migrate
```
Comment (leeky) [inline script/update:16]: Is this right? I thought the test database should use `bundle exec rails db:schema:load` as it'll truncate any static data added in the migrations?

Comment (pezholio) [reply script/update:16]: Yeah, you're probably right actually. I prefer not to add data in migrations, but this is probably safer

File: script/setup [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  echo "==> Cleaning installed Ruby dependencies..."
+  git clean -x --force -- vendor/bundle
+fi
+
+echo "==> Bootstrapping..."
+script/bootstrap
+
+if [ ! -f .env.development.local ]; then
+  echo "==> Copying default environment config..."
+  cp .env.example .env.development.local
```
Comment (leeky) [inline script/setup:20]: This is great - I've often accidentally copied this to `.env` and wondering after running the test suite where my development data has gone!

File: script/test [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+cd "$(dirname "$0")/.."
+
+if [ -n "$DEBUG" ]; then
+  set -x
+fi
+
+set +e
+echo "==> Setting up test user..."
+createuser --super test >/dev/null 2>&1
```
Comment (leeky) [inline script/test:16]: Should `DATABASE_URL` in `.env.test` be updated to use this `test` user account?

**Review by leeky — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**